pkg/cmd/server/api/types.go

```diff
@@ -83,6 +95,8 @@ const (
 	ControllersAll = "*"
 )
 
+type FeatureSet map[OpenShiftFeature]util.Empty
```

Comment:
On 3.8.2015 15:58, David Eads wrote:
> In pkg/cmd/server/api/types.go <https://github.com/openshift/origin/pull/3953#discussion_r36087108>:
>
>> @@ -83,6 +95,8 @@ const (
>>  	ControllersAll = "*"
>>  )
>>
>> +type FeatureSet map[OpenShiftFeature]util.Empty
>
> I think I'd make |FeatureSet| a |[]string| that contains a |Has| method. That way json serialization is still pretty and
> you don't lose ordering between internal and external formats. Otherwise, people will see the order of their options
> change during roundtripping.
>
> —
> Reply to this email directly or view it on GitHub <https://github.com/openshift/origin/pull/3953/files#r36087108>.
>
Does the order really matter for feature disablement?

So I can't write something like this to update master the disabled
features:

     dfs := NewStringSet(masterConfig.DisabledFeatures...)
     afs := NewStringSet(AtomicDisabledFeatures...)
     union = dfs.Union(afs)
     masterConfig.DisabledFeatures = union.List()

?